### PR TITLE
remove unexpected dot after formalpara title

### PR DESCRIPTION
**Pull request by rdmueller**
see #6562

### CODE REVIEW

**Comment by rdmueller** [comment]
note: I couldn't test the changed code yet but will do asap

File: src/Text/Pandoc/Readers/DocBook.hs [hunk 1]
```diff
@@ -708,7 +708,7 @@ parseBlock (Elem e) =
         "para"  -> parseMixed para (elContent e)
         "formalpara" -> do
            tit <- case filterChild (named "title") e of
-                        Just t  -> (para . strong . (<> str ".")) <$>
+                        Just t  -> (para . strong . (<> str " ")) <$>
```
Comment (alerque) [inline src/Text/Pandoc/Readers/DocBook.hs:711]: Why append a space? This should probably be removed entirely, maybe like:

```suggestion
                        Just t  -> (para . strong . str) <$>
```

Comment (rdmueller) [reply src/Text/Pandoc/Readers/DocBook.hs:711]: perfect - I am not a haskell developer and thought my change was on the safe side.
Your suggestion is perfect. Thanx!

Comment (rdmueller) [reply src/Text/Pandoc/Readers/DocBook.hs]: hm. seems not to work - breaks tests

**Comment by mb21** [comment]
thanks for the pull! I'm wondering though, there surely was a reason why the dot was added... aren't we breaking another case by just removing it?

**Comment by rdmueller** [comment]
that's what I wondered too. But I couldn't find a reason in the original issue or commit why the dot was appended and it also doesn't break the tests. In addition, I strongly believe that pandoc - as a converter - should not change the content. But it does in this case - not only for the docx output. I also gave it a try with HTML.

**Comment by rdmueller** [comment]
I've now checked the history of the code and can't find a reason for the dot.
But it is somehow funny that the few rendering examples in the definitive all add the dot. But I still think that this is wrong:

https://tdg.docbook.org/tdg/4.5/formalpara.html

In the current version, the title even vanished completly:

https://tdg.docbook.org/tdg/5.1/formalpara.html

**Comment by jgm** [comment]
The dot is probably there to match the rendering example.

**Comment by jgm** [comment]
Leaving the dot makes sense to me, since (a) it provides a break between the title and the rest of the paragraph, and (b) it is present in the documentation's own example.

**Comment by rdmueller** [comment]
what if we make it configurable?
In my eyes, the dot does not make sense because
- it is not part of the content
- the title is already distinguished by another format
- a colon `:` could also make sense if there is no other visual distinguished style between title and paragraph

The dot is also rendered for HTML output. In HTML, I would expect to add it through a CSS-style, if necessary.

If you could point me to an example where a command line option is used, I could try to update my PR to make the dot configurable with `.` as default. This would not break any other installations and give us the opportunity not to use a workaround.

But I would also be ok for me if you close this issue. I would then try not to use `formalpara` in my documents anymore

**Comment by jgm** [comment]
I think it's too cluttered to make this configurable with a command line option.
The way to make it configurable would be to put the title in a specially marked span, so Lua filters can manipulate it (adding or subtracting a dot).

**Comment by rdmueller** [comment]
so, I do understand that this is a "won't fix". Can you please apply the corresponding label and cose this PR and the issue?
thanx!

**Comment by jgm** [comment]
You can close the PR if you like.  But as for the issue, I think it's not quite a "wont fix" -- the course of action I think makes most sense is to enclose the title in a marked Span so that it can be manipulated by a Lua filter, as noted above.  Probably that suggestion should be added to the issue.

**Comment by nikolaialex** [comment]
I really would like to see this PR merged. The current implementation is really annoying. Especially, if there is no reasoning why the dot is added in the first place.

If you cannot merge it, maybe you can provide a filter example that fixes that. 

Thanks in advance.

**Comment by jgm** [comment]
This issue has been closed by e16df8d2718d3a2901c6ac0d7f35165755cde89f
which handles it a bit differently than the patch proposed here.
Thank you.

**Pull request closed without merging**